<launch>
  <node pkg="remote_lab" type="object_detection.py" name="aruco_tag_detection" output="screen"/>
</launch>